Userland/Libraries/LibTLS/Certificate.h

```diff
@@ -131,6 +132,8 @@ class DefaultRootCACertificates {
 
     Vector<Certificate> const& certificates() const { return m_ca_certificates; }
 
+    void reload_certificates(NonnullRefPtr<Core::ConfigFile>&);
```

Comment:
No point in taking a refptr as a reference, either take a `ConfigFile&` or a `NonnullRefPtr<ConfigFile>` (prefer the first)